packages/repository-tests/src/crud/relations/acceptance/has-many-through-inclusion-resolver.acceptance.ts

```diff
@@ -185,19 +231,27 @@ export function hasManyThroughInclusionResolverAcceptance(
         expect(toJSON(result)).to.deepEqual(toJSON(expected));
       });
 
-      it('honours limit scope when returning a model', async () => {
+      it('honours limit and order scope when returning a model', async () => {
```

Comment:
There are 3 customers of which only 2 are selected due to the `limit`. The two that are expected to be returned are Zelda and Voldemort (in that order). Since that order does not correlate to either the creation sequence of the customers nor the linking sequence of the carts to the customers, it can only be explained by successful usage of the `order` clause. (Although it could technically be caused by unexpected random order returned by the memory connnector ..)

Perhaps I should add a comment with this explanation to the test?

Reply:
I just noticed my response was about a different section of the changeset.

In this test, I added an order clause which should put the second cart item (shield) as the first of the included cart items. The last `expect` tests whether that is the case.